eZ/Publish/Core/MVC/Symfony/Security/Tests/Authentication/RememberMeRepositoryAuthenticationProviderTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+/**
+ * File containing the RememberMeRepositoryAuthenticationProviderTest class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\MVC\Symfony\Security\Tests\Authentication;
+
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\API\Repository\Values\User\User as ApiUser;
+use eZ\Publish\API\Repository\Values\User\UserReference;
+use eZ\Publish\Core\MVC\Symfony\Security\Authentication\RememberMeRepositoryAuthenticationProvider;
+use eZ\Publish\Core\MVC\Symfony\Security\User;
+use eZ\Publish\Core\Repository\Helper\LimitationService;
+use eZ\Publish\Core\Repository\Helper\RoleDomainMapper;
+use eZ\Publish\Core\Repository\Permission\PermissionResolver;
+use eZ\Publish\SPI\Persistence\User\Handler as UserHandler;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Security\Core\Authentication\Token\AnonymousToken;
+use Symfony\Component\Security\Core\Authentication\Token\RememberMeToken;
+use Symfony\Component\Security\Core\User\UserCheckerInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+class RememberMeRepositoryAuthenticationProviderTest extends TestCase
+{
+    /**
+     * @var RememberMeRepositoryAuthenticationProvider
+     */
+    private $authProvider;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|\eZ\Publish\API\Repository\Repository
+     */
+    private $repository;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->repository = $this->getMock(Repository::class);
```

Comment:
should have been createMock :|

Reply:
Fixed https://github.com/ezsystems/ezpublish-kernel/pull/2106